src/EnergyPlus/api/runtime.h

```diff
@@ -136,6 +136,14 @@ ENERGYPLUSLIB_API int energyplus(EnergyPlusState state, int argc, const char *ar
 /// \param[in] state An active EnergyPlusState instance created with `stateNew`.
 ENERGYPLUSLIB_API void stopSimulation(EnergyPlusState state);
 
+/// \brief Mutes console output (stdout and stderr) to keep terminal clear when running EnergyPlus as a library
+/// \details Normally EnergyPlus will print lots of status update messages while running.  These can get in the way
+///          when calling EnergyPlus as a library.  This function mutes the standard output messages.  Note that these
+///          can still be retrieved using callback functionality.
+/// \param[in] state An active EnergyPlusState instance created with `stateNew`.
+/// \param[in] muteOutput An integer describing whether the console output should be muted or not: nonzero integer for yes, 0 for no.
+ENERGYPLUSLIB_API void setConsoleOutputMuteState(EnergyPlusState state, int muteOutput);
```

Comment:
However, this one could use some wordsmithing.  When the function was purely there to mute the output, it was nicely named `muteConsoleOutput`.  But now that it can mute it, but also allow it to come back on, then that makes the naming a bit funny.  Should I abandon the word mute entirely and just have it be `setConsoleOutputState`?

Reply:
I agree, the "mute" isn't necessary. ```setConsoleOutputState``` is what I would prefer.